config/manager/persistent-volume-claim.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: repo-pvc
```

Comment:
probably need to rename it something like: `numaplane-repo-pvc` or `numaplane-pvc`